### PR TITLE
feat(publish): Implemented publish queue preview sidebar.

### PR DESCRIPTION
**Pull request by eozan**
SD-2375

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:  (a behave test will help)

File: client/app/scripts/superdesk-publish/views/publish-queue.html [hunk 1]
```diff
@@ -105,5 +105,28 @@ <h3 class="page-nav-title" ng-hide="multiSelectCount > 0" translate>Publish Queu
             </div> 
        </div>
     </div>
+    <div class="preview-pane content-item-preview" ng-init="tab = 'content'">
```
Comment (petrjasek) [inline client/app/scripts/superdesk-publish/views/publish-queue.html]: you can use `sd-item-preview` directive it was added in #767

**Comment by eozan** [comment]
Changed to use new sd-item-preview directive.

**Comment by ioanpocol** [comment]
:+1:

**Comment by plamut** [comment]
Seems fine, but please add the test(s), too.

**Comment by sivakuna-aap** [comment]
:+1: Might need rebase

**Comment by ioanpocol** [comment]
Per Ozan feedback he will add the tests in a different PR

**Comment by plamut** [comment]
I don't think this is the right approach, tests should be shipped along with the code so that it is always possible to automatically test a new commit. It kind of becomes tempting to postpone that separate PR with tests to "later"...

**Pull request merged**